SpatialGDK/Documentation/content/helper-scripts.md

```diff
@@ -1,15 +1,15 @@
-
+| `BuildWorker.bat` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\Build.bat` with no cooking or zipping performed.|
+| `ExternalSchemaCodegen.bat` | `<schema_directory> <output_code_directory>` | If you create [non-Unreal server-worker types]({{urlRoot}}/content/non-unreal-server-worker-types), using the [SpatialOS Worker SDK](https://docs.improbable.io/reference/13.8/shared/sdks-and-data-overview) outside of the GDK and your Unreal Engine, you manually create [schema]({{urlRoot}/content/glossary#schema). Use this script to generate Unreal code from manually-created schema; it enables your Unreal game code to interoperate with non-Unreal server-worker types. </br></br>**Example:** </br> `ExternalSchemaCodegen.bat spatial\schema\my_external_schema Game\Source\ThirdPersonShooter\ExternalSchemaCodegen` </br></br> The script forwards its arguments to a code-generator binary executable within the GDK; the binary was created by the `setup.bat` script when you first built the GDK for Unreal module dependencies.  The binary takes manually-defined schema and generates Unreal C++ types and an interface for sending and registering callbacks for worker instances to receive SpatialOS component updates.</br></br> **Flags:** </br>Both flags are defined relative to the project root.</br> * `<schema_directory>` is a subdirectory of the `spatial\schema` folder;  you manually create this folder and the schema files in it when you set up non-Unreal server-worker types - these are not schema files generated by the GDK. </br> * `<output_code_directory>` is an output directory created for the code generated by the binary and should be inside your `Source` directory.|
 <br/>
 
 <br/>------------<br/>
-_2019-06-27 Page updated with editorial review_
```

Comment:
Why did you change the style of this line? Also remember to update the date when this is approved.

Reply:
updated